src/VisualStudio/TestUtilities/OutOfProcess/CSharpInteractiveWindow_OutOfProc.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.VisualStudio.Test.Utilities.InProcess;
+
+namespace Roslyn.VisualStudio.Test.Utilities.OutOfProcess
+{
+    public class CSharpInteractiveWindow_OutOfProc : InteractiveWindow_OutOfProc<CSharpInteractiveWindow_InProc>
```

Comment:
I like the InProc decorators; I don't like that the implementation names also becomes part of an API. There's a hope that as more things need to do VS testing, more people will consume this as an opaque library. At that point we're leaking a naming convention which is useful for the implementation as contract, and that seems, well, leaky.